packages/plugins/template/readme.md

```
@@ -1,3 +1,93 @@
+
+## 模板类别和模板
+
+### 入口
+在左侧工具栏点击“模板管理”图标，进入模板管理页面。
+
+
+![alt text](./assets/entrance.png)
+![alt text](./assets/management-overview.png)
+
+### 模板类别和模板的CRUD
+**新增**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)